MailChimp.Net/Core/MailChimpException.cs

```diff
@@ -57,11 +57,9 @@ public MailChimpException(SerializationInfo info, StreamingContext context)
             }
             catch
             {
-                // ignored
             }
             finally
             {
-                Trace.Write(errorText);
                 Console.Error.WriteAsync(errorText);
```

Comment:
Why this construct? Wouldn't it be better to have error text in the exception message (in order to be able to see it when reading stacktrace et.c.)? 

I.e. doing something like:
```
    public class MailChimpException : Exception
    {
        public MailChimpException(MailChimpError error):base(FormatToString(error))
        {
              this.Detail = error.Detail;
              this.Instance = error.Instance;
...
        }
...
```
Where ```MailChimpError``` is the error deserialised from mailchimp